### PR TITLE
Add script to upload results in report portal

### PR DESCRIPTION
**Pull request by aloganat**
Signed-off-by: Arthy Loganathan <aloganat@redhat.com>

### CODE REVIEW

File: utils/scripts/reportportal/rp_uploader.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+from util import execute_command
+from logger import log
+
+"""
+Class for Report portal operations
+"""
+
+
+class reportPortalOperations():
```
Comment (tarukumar) [inline utils/scripts/reportportal/rp_uploader.py:17]: i think uppercase initial class name should be there

File: utils/scripts/reportportal/rp_uploader.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        """Parse CLI arguments"""
+
+        parser = argparse.ArgumentParser(
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            description='Script to upload results to report portal')
+
+        subparsers = parser.add_subparsers(title='Available sub commands',
+                                           help='sub-command help')
+
+        #Argument parsers for create_cluster
```
Comment (tarukumar) [inline utils/scripts/reportportal/rp_uploader.py:69]: Comment should be changed to something like  #Argument parsers for upload_rp_result

**Review by bdattoma — COMMENTED** [review]
other than @tarukumar 's comments, it looks good to me

File: utils/scripts/reportportal/rp_uploader.py (hunk 1 above)
Comment (aloganat) [reply utils/scripts/reportportal/rp_uploader.py]: Comments Incorporated

File: utils/scripts/reportportal/rp_uploader.py (hunk 2 above)
Comment (aloganat) [reply utils/scripts/reportportal/rp_uploader.py]: Comments Incorporated

**Review by bdattoma — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]
LGTM

**Pull request merged**